src/brain-even.js

```diff
@@ -0,0 +1,39 @@
+import readlineSync from 'readline-sync';
+
+console.log('Welcome to the Brain Games!');
+const userName = readlineSync.question('May I have your name? ');
+console.log(`Hello, ${userName}!`);
+
+const brainEven = () => {
+  console.log('Answer "yes" if the number is even, otherwise answer "no".');
+  const numberOfRound = 3;
+  for (let i = 0; i < numberOfRound; i += 1) {
+    const getRandomNumber = () => Math.floor(Math.random() * 1000);
+    const randomNumber = getRandomNumber();
+
+    console.log(`Question: ${randomNumber}`);
+    const answer = readlineSync.question('Your answer: ');
+
+    if (answer !== 'yes' && answer !== 'no') {
```

Comment:
This condition is not required here.

Reply:
but without this condition all answers like 'cdsd' or '12312' will be 'Correct'..

Reply:
There is no answer in the condition of the problem "Let's try again, ${userName}!". There are only answers indicating the correct answer.

Reply:
not relevant, changed the code